src/stores/AnimatorStore.ts

```diff
@@ -171,6 +192,10 @@ export class AnimatorStore {
         return 1000.0 / clamp(this.frameRate, this.minFrameRate, this.maxFrameRate);
     }
 
+    @computed get serverAnimationActive() {
+        return this.animationState === AnimationState.PLAYING && this.animationMode !== AnimationMode.FRAME;
```

Comment:
the original logic in animate() is 
this.animationState === AnimationState.PLAYING && this.animationMode === AnimationMode.FRAME
is this typo?

Reply:
good catch! you're right. Line 151 shouldn't be replaced by `serverAnimationActive`. I've updated